Assets/MixedRealityToolkit.Tests/PlayModeTests/ManipulationHandlerTests.cs

```diff
@@ -705,6 +706,7 @@ public IEnumerator ManipulationHandlerOriginOffset()
             }
             mesh.vertices = vertices;
             mesh.RecalculateNormals();
+            mesh.RecalculateBounds();
```

Comment:
This made the cube actually visible, previously you wouldn't be able to see the cube during 2nd part of test